lib/resque/worker.rb

```diff
@@ -11,6 +12,7 @@ module Resque
   # It also ensures workers are always listening to signals from you,
   # their master, and can react accordingly.
   class Worker
+    include Observable
```

Comment:
How is Observable being used here? I'm not seeing changed/notify_observers.

Reply:
Good catch. I'll get that fixed up early next week,